R/kfold.R

```diff
@@ -243,7 +243,7 @@ kfold.brmsfit <- function(x, ..., K = 10, Ksub = NULL, folds = NULL,
     ks <- match(k, Ksub)
     message("Fitting model ", k, " out of ", K)
     futures[[ks]] <- future::future(
-      .kfold_k(k), packages = "brms", seed = TRUE
+      .kfold_k(k), seed = TRUE
```

Comment:
Can you check when this was added originally? It is likely that I have initially added this for a reason. Alternatively, could you tell me with which OS you have checked that is works even without loading brms in future?

Reply:
Hm, Git Blame leads me to commit b0d9e1fb8232c95a01dbe9900324b92fd03a066e where `packages = "brms"` already existed. I don't manage to track it down further than to that commit.

I have run the reprex from my [initial PR comment](https://github.com/paul-buerkner/brms/pull/1287#issue-1116083279) on Windows 10 and Ubuntu 20.04.3 LTS. On the same systems, I have run the brms `R CMD check`.

Reply:
Ok, thank you!